CMakeLists.txt

```diff
@@ -428,10 +428,17 @@ else() #!WIN32
     if (QUIC_OPTIMIZE_LOCAL AND NOT CMAKE_SYSTEM_PROCESSOR STREQUAL arm)
         set(MARCH -march=native)
     endif()
-    set(CMAKE_C_FLAGS_DEBUG "-Og -ggdb3")
-    set(CMAKE_C_FLAGS_MINSIZEREL "-Os -DNDEBUG")
-    set(CMAKE_C_FLAGS_RELWITHDEBINFO "-O3 ${MARCH} -ggdb3 -DNDEBUG")
-    set(CMAKE_C_FLAGS_RELEASE "-O3 ${MARCH} -DNDEBUG")
+    if (CX_PLATFORM STREQUAL "darwin")
+        set(CMAKE_C_FLAGS_DEBUG "-Og -gdwarf")
```

Comment:
Looks like this is the only flag that's different? Why duplicate all the others?